canonical-web-and-design/jaas-dashboard
# Update deps

## Done

Update old dependencies.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- The app should build and function as expected.

## Details

Fixes #266
